N: 720

max_speed: 0.40
min_speed: 0.0
max_yawrate: 1.0
max_accel: 4.0
max_dyawrate: 1.5         
 
v_reso: 0.05
yawrate_reso: 0.05

dt: 0.1
predict_time: 3.0

to_goal_cost_gain: 2.0
speed_cost_gain: 0.3
obstacle_cost_gain: 0.2

robot_radius: 0.25

roomba_v_gain: 0.4
roomba_omega_gain: 0.4
                        
